src/Adapter/Feature/CombinationFeature.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * 2007-2017 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2017 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Adapter\Feature;
+
+use Combination;
+use PrestaShop\PrestaShop\Adapter\Configuration;
+
+/**
+ * This class manages Combination feature
+ */
+class CombinationFeature
```

Comment:
You made all the classes inside this namespace with the same structure, it would be great to write an interface to make sure we will follow you with the other potential features.

Reply:
yes, it's in my internal brain todo :)

Reply:
:ok_hand:

Reply:
done